packages/wonder-blocks-dropdown/src/components/check.tsx

```diff
@@ -16,30 +14,17 @@ type CheckProps = {
     disabled: boolean;
     /** Whether option item is selected. */
     selected: boolean;
-    /** Whether option item is pressed. */
-    pressed: boolean;
-    /** Whether option item is hovered. */
-    hovered: boolean;
-    /** Whether option item is focused. */
-    focused: boolean;
 };
 
 /**
  * The check component used by OptionItem.
  */
 const Check = function (props: CheckProps): React.ReactElement {
-    const {disabled, selected, pressed, hovered, focused} = props;
+    const {selected} = props;
     return (
         <PhosphorIcon
             icon={checkIcon}
             size="small"
-            color={
-                disabled
-                    ? offBlack32
-                    : pressed || hovered || focused
-                    ? white
-                    : offBlack
-            }
```

Comment:
**note:** Removed this as we no longer depend on `Clickable/getClickableBehavior` and instead we assign this with CSS (defined in `OptionItem`).